build.gradle

```diff
@@ -1,229 +1,388 @@
+        } else {
+            jdbc_driver_class = "org.postgresql.Driver"
+            jdbc_dialect = "org.hibernate.dialect.PostgreSQLDialect"
+            jdbc_quartz_driver_class = "org.quartz.impl.jdbcjobstore.PostgreSQLDelegate"
+            jdbc_url = "jdbc:postgresql://${db_host}/${db_name}"
+        }
+
+        amqp_enabled = "true".equals(project.findProperty("qpid"))
+        external_broker = "true".equals(project.findProperty("external_broker"))
+        hostedtest = "true".equals(project.findProperty("hostedtest"))
     }
-    // IF MYSQL set up the mysql stuff
-    // else set up postgres (default)
-    if (project.findProperty('database_server') == 'mysql') {
-      jdbc_driver_class = 'com.mysql.jdbc.Driver'
-      jdbc_dialect = 'org.hibernate.dialect.MySQL5InnoDBDialect'
-      jdbc_quartz_driver_class = 'org.quartz.impl.jdbcjobstore.StdJDBCDelegate'
-      jdbc_url = "jdbc:mysql://${db_host}/${db_name}"
-    } else {
-      jdbc_driver_class = 'org.postgresql.Driver'
-      jdbc_dialect = 'org.hibernate.dialect.PostgreSQLDialect'
-      jdbc_quartz_driver_class = 'org.quartz.impl.jdbcjobstore.PostgreSQLDelegate'
-      jdbc_url = "jdbc:postgresql://${db_host}/${db_name}"
+
+    checkstyle {
+        configFile = file("${rootProject.rootDir}/project_conf/checks.xml")
+        configProperties = [
+            "checkstyle.header.file": file("${rootProject.rootDir}/project_conf/HEADER.txt").getCanonicalPath(),
+            "javadoc.type.scope": "package",
+            "javadoc.method.scope": "public",
+            "javadoc.lazy": "false",
+            "javadoc.missing.param": "true",
+            "javadoc.var.scope": "public",
+            "checkstyle.cache.file": file("${project.buildDir}/cache/checkstyle.cache.src").getCanonicalPath()]
     }
 
-    amqp_enabled = 'true'.equals(project.findProperty("qpid"))
-    external_broker = 'true'.equals(project.findProperty('external_broker'))
-    hostedtest = 'true'.equals(project.findProperty('hostedtest'))
-  }
-
-
-  checkstyle {
-    configFile = file("${rootProject.rootDir}/project_conf/checks.xml")
-    configProperties = [
-            'checkstyle.header.file': file("${rootProject.rootDir}/project_conf/HEADER.txt").getCanonicalPath(),
-            'javadoc.type.scope': 'package',
-            'javadoc.method.scope': 'public',
-            'javadoc.lazy': 'false',
-            'javadoc.missing.param': 'true',
-            'javadoc.var.scope': 'public',
-            'checkstyle.cache.file': file("${project.buildDir}/cache/checkstyle.cache.src").getCanonicalPath()]
-  }
-
-  dependencies {
-    checkstyle 'com.puppycrawl.tools:checkstyle:8.2'
-    checkstyle project(':checks')
-  }
+    dependencies {
+        checkstyle libraries.checkstyle
+        checkstyle project(":checks")
+    }
 }
+
+task pom {
+    doLast {
+        // Exclude all transitive dependencies
+        configurations.all*.exclude module: "*"
+
+        pom {
```

Comment:
We don't need the top-level pom to be generated by Gradle.  We've been maintaining it by hand since the beginning.  The top-level pom almost never changes since it's just plugin version and project meta-data.  Please remove this task because I'd like to continue maintaining the top-level pom by hand.

Reply:
Done.